### PR TITLE
rbd/bench: add notes of default values, it's easy to use

### PR DESCRIPTION
**Pull request by zy751713126**
Signed-off-by: Zheng Yin <zhengyin@chinac.com>

### CODE REVIEW

File: src/test/cli/rbd/help.t [hunk 1]
```diff
@@ -121,9 +121,9 @@
   Optional arguments
     -p [ --pool ] arg    pool name
     --image arg          image name
-    --io-size arg        IO size (in B/K/M/G/T)
-    --io-threads arg     ios in flight
-    --io-total arg       total size for IO (in B/K/M/G/T)
+    --io-size arg        IO size (in B/K/M/G/T) [default: 4096B]
```
Comment (trociny) [inline src/test/cli/rbd/help.t]: nit: may be '4K'?

File: src/test/cli/rbd/help.t [hunk 2]
```diff
@@ -121,9 +121,9 @@
...
+    --io-threads arg     ios in flight [default: 16]
+    --io-total arg       total size for IO (in B/K/M/G/T) [default: 1G]
     --io-pattern arg     IO pattern (rand or seq)
     --io-type arg        IO type (read or write)
```
Comment (trociny) [inline src/test/cli/rbd/help.t]: I would add default descriptions for IO  pattern and IO type too.

File: src/test/cli/rbd/help.t (hunk 1 above)
Comment (zy751713126) [inline src/test/cli/rbd/help.t]: yes

File: src/test/cli/rbd/help.t (hunk 2 above)
Comment (zy751713126) [inline src/test/cli/rbd/help.t]: Default description for IO pattern is seq, I will add it. But IO type has no default description.

File: src/test/cli/rbd/help.t (hunk 1 above)
Comment (zy751713126) [inline src/test/cli/rbd/help.t]: yes

File: src/test/cli/rbd/help.t (hunk 2 above)
Comment (trociny) [inline src/test/cli/rbd/help.t]: Agreed. I thought it was write by default.

**Comment by yangdongsheng** [comment]
Looks good.

**Pull request merged**